corehq/apps/app_manager/commcare_settings.py

```diff
@@ -99,9 +103,7 @@ def get_custom_commcare_settings():
 
 @memoized
 def get_commcare_settings_layout(doc_type):
-    if doc_type == "LinkedApplication":
-        return {}
-    if doc_type in ['Application', 'RemoteApp']:
+    if doc_type in ['Application', 'RemoteApp', 'LinkedApplication']:
```

Comment:
It'd be cleaner to use [app_doc_types](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/util.py#L63) - should be fine to check both the base types and the deleted types, or just the base types, doesn't matter.